packages/rpc/src/methods/stacks/stx-transfer-sip9-nft.ts

```diff
@@ -0,0 +1,20 @@
+import { z } from 'zod';
+
+import { defineRpcEndpoint } from '../../rpc/schemas';
+import {
+  baseStacksTransactionConfigSchema,
+  stacksTransactionDetailsSchema,
+} from './_stacks-helpers';
+
+export const stxTransferSip9Nft = defineRpcEndpoint({
+  method: 'stx_transferSip9Nft',
+  params: z.intersection(
+    z.object({
+      recipient: z.string(),
+      asset: z.string(),
+      assetId: z.string(),
```

Comment:
Yep, looks good, since wallets right now use serialized clarity-values. In the future this `assetId` could also be the Clarity value json structure.

Reply:
I see the `id` in the confirmed tx now, so nvm here.